### PR TITLE
Assingment python day 3 MVP

### PR DESCRIPTION
**Pull request by fuse999**

### CODE REVIEW

File: src/adv.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
+import time
```
Comment (AdnanCodes) [inline src/adv.py]: Make more commits! I know you are new to this, break every functional code into individual commits. Showcasing how you built your code.

File: src/adv.py [hunk 2]
```diff
@@ -38,14 +39,47 @@
...
+# player = input("What is your name, brave soul : "), room['outside']
+# print(player)
 # Write a loop that:
-#
-# * Prints the current room name
+# player_cmd = ""
+# while player_cmd not in ():
+while True:
+    # * Prints the current room name
+    time.sleep(1)
```
Comment (AdnanCodes) [inline src/adv.py]: Comment on why you decided to use time.sleep

File: src/adv.py [hunk 3]
```diff
@@ -38,14 +39,47 @@
...
+            player.current_room = current_room.w_to
+        else:
+            print("Can't go that way silly")
+        # If the user enters "q", quit the game.
+    elif move == "q":
+        print("Thank you for playing! Good Bye!")
+        exit()
+    else:
+        # Print an error message if the movement isn't allowed.
+        print("Not a valid move. Please enter: n, s, e, w or q")
```
Comment (AdnanCodes) [inline src/adv.py]: Can you refactor all the logic into a more compact code?

**Review by AdnanCodes — COMMENTED** [review]
Great Work so far, you are on point!

File: src/adv.py (hunk 2 above)
Comment (fuse999) [reply src/adv.py]: adds suspense??? more experimenting

File: src/adv.py (hunk 3 above)
Comment (fuse999) [reply src/adv.py]: yes

**Pull request merged**